### PR TITLE
Fixes #2882: Package a SELinux policy for rudder-webapp

### PR DESCRIPTION
**Pull request by Kegeruneku**
Ticket: http://www.rudder-project.org/redmine/issues/2882

To be merged in 3.1

### CODE REVIEW

File: rudder-webapp/debian/dirs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 opt/rudder/bin
 opt/rudder/share/webapps
 opt/rudder/share/rudder-plugins
+opt/rudder/share/certificates
```
Comment (peckpeck) [inline rudder-webapp/debian/dirs]: What is this directory doing here ?

**Comment by peckpeck** [comment]
OK, this is a good PR

File: rudder-webapp/debian/dirs (hunk 1 above)
Comment (Kegeruneku) [inline rudder-webapp/debian/dirs]: It has been added to be consistent with the SPEC file (and it corrects a small mistake, this dir belongs to the rudder-webapp package and had been forgotten when we added cert management)

**Comment by peckpeck** [comment]
OK Good

**Pull request merged**